artifacts/definitions/Windows/Registry/AppCompatCache.yaml

```diff
@@ -16,6 +16,6 @@ sources:
               SELECT Filename, Data FROM read_file(
                   filenames=AppCompatKeys.FullPath, accessor='reg')
           }, query={
-              SELECT Filename AS Key, name as Name, epoch, time
+              SELECT Filename AS Key, name as Name, epoch, ModificationTime
```

Comment:
I think you will need to make this: time AS ModificationTime

Reply:
Yep, my bad, will fix